### PR TITLE
Database library

### PR DESCRIPTION
**Pull request by nickpezzotti1**
The query functions should be specific for the use - But this is fine as an example.

e.g. insert is too generic

### CODE REVIEW

**Comment by LeithanK** [comment]
Yes, for now it's really basic. In the future we would be looking at only specifying some arguments and also sanity check the arguments.

File: README.md [hunk 1]
```diff
@@ -16,3 +16,13 @@ Buddy scheme management system web application.
 
 1. `$ source env/bin/activate`
 2. `(env) $ python app.py`
+
+## Datababse credentials
```
Comment (nickpezzotti1) [inline README.md]: Database*

File: README.md [hunk 2]
```diff
@@ -16,3 +16,13 @@ Buddy scheme management system web application.
...
+To set the variables permanently, you should edit your `.bashrc` in your home directory.
+
+1. `(env) $ export BUDDY_DB_USER='admin'`
+2. `(env) $ export BUDDY_DB_PASSWORD='r5!!eXFNVC5qMDL3$o&m'`
+
+## Connect to databse
```
Comment (nickpezzotti1) [inline README.md]: Again

File: flaskr/lib/database/basic.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+
+def query(sql_query):
+    """ Returns results of query """
+
+    conn = pymysql.connect(DB_HOST, user=DATABASE_USER, password=DATABASE_PASSWORD, db=DB_NAME, connect_timeout=5)
+   
+    try:
+    
+        # Select all the users that are in the french society and don't have a ticket yet
```
Comment (nickpezzotti1) [inline flaskr/lib/database/basic.py]: remove this comment

**Comment by nickpezzotti1** [comment]
Before we merge we need at least a few mentees and mentors in the database

**Comment by nickpezzotti1** [comment]
Is there a way to view the current state of the database? Perhaps seeing all the tables currently in the database and all the entries?

**Comment by LeithanK** [comment]
I'll look up a tool. I'm sure there's one.
On top of this, I think it's better for us to merge once we created more functions, namely:

- Insert into students
- Modify students based on K number or other attributes
- Delete entries

This has to be done for each of the table, as well as a sanity check for the input, ie no " "" ); drop table;"

**Pull request closed without merging**